packages/cli/src/commands/init-flow.ts

```diff
@@ -7,64 +6,68 @@ import ora from "ora";
 import { link } from "./link";
 import { sync } from "./sync";
 import { build, buildOptions } from "./build";
+import { prompt } from "../prompts";
 import type { StrictYargsOptionsToInterface } from "./yargs-types";
 
 export const initFlow = async (
   options: StrictYargsOptionsToInterface<typeof buildOptions>
 ) => {
   const isProjectConfigured = await isFileExists(".webstudio/config.json");
-  const prompsList: PromptObject[] = [];
-  const spinner = ora().start();
+  let shouldInstallDeps = false;
 
   if (isProjectConfigured === false) {
-    prompsList.push(
-      {
-        type: "confirm",
-        name: "folder",
-        message: "Do you want to create a folder",
-        initial: true,
-      },
-      {
-        type: (prev) => (prev === true ? "text" : null),
+    const { shouldCreateFolder } = await prompt({
+      type: "confirm",
+      name: "shouldCreateFolder",
+      message: "Do you want to create a folder",
+      initial: true,
+    });
+
+    if (shouldCreateFolder === true) {
+      const { folderName } = await prompt({
+        type: "text",
         name: "folderName",
         message: "Enter a project name",
-      },
-      {
-        type: (prev) => (prev === undefined ? null : "text"),
-        name: "projectLink",
-        message: "Enter a project link",
+      });
+
+      if (folderName === undefined) {
+        throw new Error("Folder name is required");
```

Comment:
why? if they don't enter it, it goes to "./" like we discussed

Reply:
It does go, this is if they selected the option to create a folder and still don't enter 👍